docs/stdlib/array.rst

```diff
@@ -197,7 +194,9 @@ Reference
 
 .. eql:operator:: arrayplus: array<anytype> ++ array<anytype> -> array<anytype>
 
-    Array concatenation.
+    Concatenates given arrays of :eql:type:`anytype` into one.
+
+    This results in a reference of both array's elements conjoined together:
```

Comment:
This seems like it would be easy to misinterpret. Maybe this instead?
> This results in an array containing the elements of both of the concatenated arrays.